GetIntoTeachingApi/Models/TeacherTrainingAdviserSignUp.cs

```diff
@@ -164,6 +178,25 @@ private Candidate CreateCandidate()
             return candidate;
         }
 
+        private string SanitizeTelephone(string telephone)
+        {
+            var international = CountryId != LookupItem.UnitedKingdomCountryId;
+
+            if (international && telephone != null)
+            {
+                // Remove non-digit characters.
+                telephone = Regex.Replace(telephone, "[^0-9]", string.Empty);
+
+                // Prefix with 00.
+                telephone = $"00{telephone}";
```

Comment:
Is it safe to assume that the person who typed it didn't already prepend the exit code?

Reply:
Yeah we validate on the front-end to ensure the phone number starts with the country dial-in code

Reply:
I didn't know that was called an exit code though - I'll update the comment 😄